go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/xanzy/go-gitlab v0.81.0
 	go.etcd.io/bbolt v1.3.7
 	go.uber.org/zap v1.24.0
+	golang.org/x/exp v0.0.0-20230321023759-10a507213a29
```

Comment:
We used to have this and removed it. Let's not bring this back as it can be a pain in maintenance since it's for deprecated functions and it's not tagged so there are rapid releases leading to many renovatebot prs.

Reply:
I'll update it

Reply:
@jukie see below for suggested changes

Reply:
Fixed, thanks!